spreadsheets/intermediate/charts-2/example-use-case-for-charts.md

```diff
@@ -0,0 +1,50 @@
+---
+author: Stefan-Stojanovic
+
+aspects:
+  - introduction

```

Comment:
since we're in the intermediate topic, I don't think the `introduction` aspect is fitted here